target/plantbattles-1/index.html

```diff
@@ -3,10 +3,34 @@
   <head>
     <meta charset="UTF-8">
     <title>Plant Battles</title>
+
+    <link rel="icon" href="/images/plantFav.png" type="image/x-icon">
+    <link href="https://fonts.googleapis.com/css2?family=PT+Mono&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
```

Comment:
do we have this file in here twice? it's also webapp/index.html. We probably only need one copy of this.